pkgs/top-level/perl-packages.nix

```diff
@@ -6091,6 +6091,13 @@ let
     patches = [
       # Support building against our own sqlite.
       ../development/perl-modules/DBD-SQLite/external-sqlite.patch
+
+      # Pull upstream fix for test failures against sqlite-3.37.
+      (fetchpatch {
```

Comment:
Can we update to https://github.com/DBD-SQLite/DBD-SQLite/releases/tag/1.71_01 instead?

Reply:
The perl updates pull from cpan, not github